tests/functional/c/consider/consider_using_augmented_assign.py

```diff
@@ -0,0 +1,77 @@
+"""Tests for consider-using-augmented-assign."""
+
+# pylint: disable=invalid-name,too-few-public-methods,import-error,consider-using-f-string
+
+from unknown import Unknown
+
+x = 1
+x = x + 1  # [consider-using-augmented-assign]
+x = 1 + x  # [consider-using-augmented-assign]
+x, y = 1 + x, 2 + x
```

Comment:
Shouldn't the left part of the tuple raise ?

Reply:
To what?

```python
x += 1
y = 2 + x
```

I don't think that's nicer, or at least too opinionated for a check which is more subtle like the current one.

Reply:
Right, I did not consider that.